tests/tracer/test_logging.py

```diff
@@ -0,0 +1,518 @@
+import os
+import re
+
+import pytest
+
+from ddtrace.internal.compat import PY2
+
+
+LOG_PATTERN = r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3} \w{1,} \[\S{1,}\] \[\w{1,}.\w{2}:\d{1,}\] - .{1,}$"
```

Comment:
Can you add a comment to describe this `LOG_PATTERN`. One or two examples of a matched string would be helpful. 

Also is there a reason why we're using a regex instead of asserting that a string was logged?

Reply:
An example log of this format that this regex is looking for:

```
'2022-06-10 21:49:26,010 CRITICAL [ddtrace] [test.py:15] - ddtrace critical log\n'
```

Is this the kind of example you want in the comments? I can add this to test file if it's useful!

I wrote tests to test logs recorded by the ddtrace logger with a regex for two main reasons:
1. To show that the log was created actually created in the file
2. To show that the log will follow the expected format (if it didn't, then it'll make parsing the files much more difficult)